flow/project.py

```diff
@@ -4207,18 +4210,22 @@ def foo(job):
         options : str
             A string to append to submissions. Can be any valid
             :meth:`FlowOperation.run` option. (Default value = "")
+        aggregator_obj : :class:`~.aggregator`
+            aggregator object associated with the :class:`FlowGroup`. If None, the
```

Comment:
```suggestion
            An instance of :class:`~flow.aggregator` to associate with the:class:`FlowGroup`. If None, the
```

aggregator object seems like a vague term, this feels more clear.

Reply:
Seems fine to me.